llama_stack/providers/utils/telemetry/tracing.py

```diff
@@ -180,7 +180,7 @@ async def start_trace(name: str, attributes: dict[str, Any] = None) -> TraceCont
 
     trace_id = generate_trace_id()
     context = TraceContext(BACKGROUND_LOGGER, trace_id)
-    attributes = {marker: True for marker in ROOT_SPAN_MARKERS} | (attributes or {})
+    attributes = {marker: True for marker in ROOT_SPAN_MARKERS} | {"__local_root_span__": True} | (attributes or {})
```

Comment:
Just a nit: for our future selves, we may wonder why there's this extra `__local_root_span__` here, so perhaps a brief comment about how this gets used and why it's not just part of `ROOT_SPAN_MARKERS` would help us remember the significance of this.

And, in that same vein, perhaps a constant just so we don't have to match the strings up both here and in sqlite_span_processor?